member_type.go

```diff
@@ -0,0 +1,11 @@
+package zoom
+
+// Member represents an group member
+type Member struct {
+	ID        string `json:"id"`
+	Email     string `json:"email"`
+	FirstName string `json:"first_name"`
+	LastName  string `json:"last_name"`
+	Type      int64  `json:"type"`
+	AddedAt   string `json:"added_at"`
```

Comment:
According to the api doc you linked, the only fields populated here are ID and email - where are you getting the other fields from?

Reply:
Sorry, I'm referenced other api doc. Fix it.